spring-secure-resource-server/templates/appSSOInstance.yaml

```diff
@@ -0,0 +1,111 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:yaml", "yaml")
+
+---
+apiVersion: sso.apps.tanzu.vmware.com/v1alpha1
+kind: AuthServer
+metadata:
+  name: #@ data.values.appSSOName
+  namespace: #@ data.values.workloadNamespace
+  labels:
+    name: appsso-postal-codes
```

Comment:
I think this should be 
```
     name: #@ data.values.appSSOName
```
in order to accommodate this changed via the option

Reply:
Good catch.